addons/project/views/project_views.xml

```diff
@@ -1542,6 +1498,18 @@
             </field>
         </record>
 
+        <record id="view_task_tree2_inherited" model="ir.ui.view">
```

Comment:
Can't we just do this in the main view?
`<field name="project_id" ... invisible="context.get('default_project_id', False)"/>`

Reply:
Yes it'll work, but when user reload page then project field will again display so I think we should avoid `invisible="context.get('default_project_id', False)` when it's possible
what do you think?
Thanks :)

Reply:
I might be overlooking something, but when I test in local, reloading doesn't lose the information.

Reply:
Thank you!